src/views/EdgeApplicationsRulesEngine/FormFields/FormFieldsEdgeApplicationsRulesEngine.vue

```diff
@@ -599,12 +777,29 @@
         v-for="(behaviorItem, index) in behaviors"
         :key="behaviorItem.key"
       >
-        <Divider
-          align="left"
-          type="dashed"
-        >
-          {{ behaviorItem.isFirst ? 'Then' : 'And' }}
-        </Divider>
+        <div class="flex items-center gap-2">
+          <Divider
+            align="left"
+            type="dashed"
+          >
+            {{ behaviorItem.isFirst ? 'Then' : 'And' }}
```

Comment:
criar um função que recebe isFirst e retorna Then ou And, com nomenclatura que ajude a compreender melhor o código.